torch/distributions/lowrank_multivariate_normal.py

```diff
@@ -169,8 +169,11 @@ def precision_matrix(self):
 
     def rsample(self, sample_shape=torch.Size()):
         shape = self._extended_shape(sample_shape)
-        eps_W = self.loc.new_empty(shape[:-1] + (self.cov_factor.size(-1),)).normal_()
-        eps_D = self.loc.new_empty(shape).normal_()
+        zero = self.loc.new_tensor(0.)
+        one = self.loc.new_tensor(1.)
+        W_shape = shape[:-1] + self.cov_factor.shape[-1:]
+        eps_W = torch.normal(zero.expand(W_shape), one.expand(W_shape))
```

Comment:
nit: can we use the `torch.zeros(W_shape, dtype=..,device=..)` idiom here (and other variants of normal) instead, to avoid non contiguous parameters, when possible?